content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -341,21 +341,57 @@ Some values of an object are typically generated before the object is persisted.
+This model of specifying intent makes it difficult to remove existing fields.
+When a field is removed from one's config and applied, the value will be kept
+(the system assumes that you don't care about that value anymore). If an item is
+removed from a list or a map, it will be removed if no other appliers care about
+its presence.
 
-For a user to manage a field, in the Server Side Apply sense, means that the user relies on and expects the value of the field not to change. The user who last made an assertion about the value of a field will be recorded as the current field manager. This can be done either by changing the value with `POST`, `PUT`, or non-apply `PATCH`, or by including the field in a config sent to the Server Side Apply endpoint. Any applier that tries to change the field which is managed by someone else will get its request rejected (if not forced, see the Conflicts section below).
+Changes to an object's fields are tracked through a "[field management](#field-management)"
+mechanism. When a field's value changes, ownership moves from its current
+manager to the manager making the change. When trying to apply an object, fields
+that have a different value and are owned by another manager will result in a
+[conflict](#conflicts). This is done in order to signal that the operation might undo another
+collaborator's changes. Conflicts can be forced, in which case the value will be
+overriden, and the ownership will be transfered.
 
-Field management is stored in a newly introduced `managedFields` field that is part of an object's [`metadata`](/docs/reference/generated/kubernetes-api/v1.14/#objectmeta-v1-meta).
+It is meant both as a replacement for the original `kubectl apply` and as a
+simpler mechanism to write controllers.
+
+### Field Management
+
+Compared to the `last-applied` annotation managed by `kubectl`, Server Side
+Apply uses a more declarative approach, which tracks a user's field management,
+rather than a user's last applied state. This means that as a side effect of
+using Server Side Apply, information about which field manager manages each
+field in an object also becomes available.
+
+For a user to manage a field, in the Server Side Apply sense, means that the
+user relies on and expects the value of the field not to change. The user who
+last made an assertion about the value of a field will be recorded as the
+current field manager. This can be done either by changing the value with
+`POST`, `PUT`, or non-apply `PATCH`, or by including the field in a config sent
+to the Server Side Apply endpoint. Any applier that tries to change the field
+which is managed by someone else will get its request rejected (if not forced,
```

Comment:
`Any applier that tries...` could need the explanation that there is apply and non-apply not explicitly before to make it more clear that only apply causes conflicts.
If somebody without knowledge of the different operations reads this, it might cause confusion.
Maybe a link to the respective section below is enough though.

Reply:
The sentence right above that sorts of create the distinction between SSA and non-SSA:
> This can be done either by changing the value with
`POST`, `PUT`, or non-apply `PATCH`, or by including the field in a config sent
to the Server Side Apply endpoint.

> We could rephrase to: When using server-side apply, trying to change the field ...

making a reference to the case mentioned right before?

Reply:
Updated, I've also added a link to the Conflicts section

Reply:
Looks good to me.
/lgtm
/approve